### PR TITLE
Add HTTP Port Option

### PR DESCRIPTION
**Pull request by Vhex**
Add the ability to process options.httpPort as part of the http/https routine.
Fix what appears to be a typo on message.data?.text?

This is related to the issue on hubot-matteruser

### CODE REVIEW

File: src/client.coffee [hunk 1]
```diff
@@ -221,7 +221,7 @@ class Client extends EventEmitter
                 @emit 'new_user', message
             else
                 # Check for `pong` response
-                if message.data?.text? and message.data.text == "pong"
+                if message.data.text? and message.data.text == "pong"
```
Comment (loafoe) [inline src/client.coffee]: @Vhex this is not a typo but an extra check to prevent crashes in case the message also does not have data object

Comment (loafoe) [inline src/client.coffee]: @Vhex I'll happily merge the httpPort stuff, also in github-matteruser but please revert the typo changeset, or split it out in a separate PR if you like to debate it :)

Comment (Vhex) [inline src/client.coffee]: @loafoe Would you prefer to just close this pull request and I'll submit a new one or is reverting it in the changeset fine?

Comment (loafoe) [inline src/client.coffee]: @Vhex already merged :)

**Pull request merged**